Makefile

```diff
@@ -226,6 +244,23 @@ OPM = $(shell which opm)
 endif
 endif
 
+.PHONY: operator-sdk
+operator-sdk: ## Download operator-sdk locally if necessary.
+	$(call operator-framework-tool, $(OPERATOR_SDK), $(OPERATOR_SDK_DIR),github.com/operator-framework/operator-sdk/releases/download/$(OPERATOR_SDK_VERSION)/operator-sdk_$${OS}_$${ARCH})
+
+# operator-framework-tool will delete old package $2, then dowmload $3 to $1.
+define operator-framework-tool
```

Comment:
How about [`url-install-tool`]( https://github.com/medik8s/fence-agents-remediation/pull/3#discussion_r1081467799)?

Reply:
I agree, but `operator-framework-tool` name can be misleading as it is a tool from the operator-framework... I don't feel strongly to which direction, your choice.

Reply:
If we prefer to stick with **operator-framework** prefix then _operator-framework-tool_ or _operator-framework-install-tool_ makes more sense. I tend to go with just `operator-framework-tool`.